Resources/doc/reference/action_list.rst

```diff
@@ -192,6 +194,10 @@ Available types and associated options
 
 If you have the SonataDoctrineORMAdminBundle installed, you have access to more field types, see `SonataDoctrineORMAdminBundle Documentation <https://sonata-project.org/bundles/doctrine-orm-admin/master/doc/reference/list_field_definition.html>`_.
 
+.. note::
+
+    It is better to prefer non negative notions when possible for boolean values
```

Comment:
"... so use the `inverse` option if you really cannot find a good enough antonym for the name you have.